docs/release/overlays.md

```diff
@@ -189,4 +189,4 @@ When it comes to creation of overlays there are no strict rules on how overlays
 
     2. If a critical bug fix involves multiple components (driver and sidecar), cherry-pick all the changes to the older release branch.
 
-   After a change has been cherry-picked to a release branch, a new patch release will have to be cut on that branch, and the overlays would be updated to reflect the new driver image version.
\ No newline at end of file
+   After a change has been cherry-picked to a release branch, a new patch release will have to be cut on that branch, and the overlays would be updated to reflect the new driver image version.
```

Comment:
I don't see the change here

Reply:
Actually It is taking automatically
I was trying to make a new PR but the same thing is happening
I dont know what is the reason here